legacy/components/TxHistory/TxDetails.js

```diff
@@ -313,8 +313,14 @@ const TxDetails = () => {
           </View>
         </View>
       </Screen>
-      {/* $FlowFixMe TODO: index does not exist in AddressModal props */}
-      <AddressModal visible={!!addressDetail} onRequestClose={hideAddressModal} address={addressDetail} index={null} />
+      {addressDetail && (
+        <AddressModal
+          visible
+          onRequestClose={hideAddressModal}
+          address={addressDetail}
+          onAddressVerify={hideAddressModal}
```

Comment:
note: it is reserved for HW wallets, but is not implemented, we will be changing it on the next refactor, keeping it like this for now.